google/cloud/spanner/internal/connection_impl_test.cc

```diff
@@ -2512,6 +2519,34 @@ TEST(ConnectionImplTest, CommitSuccessWithStats) {
   EXPECT_EQ(42, commit->commit_stats->mutation_count);
 }
 
+TEST(ConnectionImplTest, CommitSuccessWithMaxCommitDelay) {
+  auto mock = std::make_shared<spanner_testing::MockSpannerStub>();
+  auto db = spanner::Database("placeholder_project", "placeholder_instance",
+                              "placeholder_database_id");
+  EXPECT_CALL(*mock, BatchCreateSessions(_, HasDatabase(db)))
+      .WillOnce(Return(MakeSessionsResponse({"test-session-name"})));
+  google::spanner::v1::Transaction txn = MakeTestTransaction();
+  EXPECT_CALL(*mock, BeginTransaction).WillOnce(Return(txn));
+  EXPECT_CALL(
+      *mock,
+      Commit(_, AllOf(HasSession("test-session-name"),
+                      HasMaxCommitDelay(std::chrono::milliseconds(100)))))
+      .WillOnce(Return(MakeCommitResponse(
+          spanner::MakeTimestamp(std::chrono::system_clock::from_time_t(123))
+              .value(),
+          spanner::CommitStats{42})));
```

Comment:
Please remove the `CommitStats` from this test.

Reply:
Done.